docs/4.x/tagmanager/integration-plugin.md

```diff
@@ -15,10 +15,12 @@ The easiest way to embed a container without needing any authentication is to as
 ```html
 <!-- Matomo Tag Manager -->
 <script type="text/javascript">
-window._mtm = window._mtm || [];
-window._mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
-var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
-g.type='text/javascript'; g.async=true; g.src='${MATOMOURL}/js/container_${CONTAINERID}.js'; s.parentNode.insertBefore(g,s);
+  window._mtm = window._mtm || [];
+  window._mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+  (function() {
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.src='https://{$MATOMO_URL}/js/container_{$CONTAINER_ID}.js'; s.parentNode.insertBefore(g,s);
+  })();
```

Comment:
The reference to `{$CONTAINERID}` a few lines below need to be updated as well.

Reply:
Thank you @sgiehl . I will make things more consistent.